lib/gc.cpp

```diff
@@ -51,24 +59,29 @@ void *operator new[](std::size_t size) {
 void operator delete(void *p) _GLIBCXX_USE_NOEXCEPT { return gc::operator delete(p); }
 void operator delete[](void *p) _GLIBCXX_USE_NOEXCEPT { return gc::operator delete(p); }
 
-extern "C" void (*GC_start_call_back)(void);
-extern "C" size_t GC_get_heap_size(void);
+#ifndef NO_GC_PRINT_STATS
```

Comment:
Would you mind avoiding double negation? That is, you can use cmakedefine to check for the GC_print_stats function, define HAS_GC_PRINT_STATS if it exists, and then use that to factor out the code. Then, you will not have any platform dependent code in your source.